src/etl/adapter/yd_parser.py

```
@@ -0,0 +1,122 @@
+                attached_files.append(f"media_type: {title.get_text(strip=True)}")
+            return MessageMetadata(
+                chat_id=self.source_name,
+                sender_id=sender_id,
+                text=text,
+                attached_files=attached_files
+            )
+
+    async def parse_batch(self, html_content: str) -> List[MessageMetadata]:
+            try:
+                soup = BeautifulSoup(html_content, 'lxml')
+            except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)